examples/spaceships/Cargo.toml

```diff
@@ -20,7 +20,7 @@ bevygap_server = ["lightyear_examples_common/bevygap_server", "server"]
 steam = ["lightyear_examples_common/steam"]
 
 [dependencies]
-lightyear_examples_common.workspace = true
+lightyear_examples_common = { workspace = true, features = ["websocket"] }
```

Comment:
that shouldn't be necessary since the `default` features above contain websocket, no?
```
default = ["lightyear_examples_common/websocket"]
```

Reply:
not necessary to run the example with cargo, but when you run the build_wasm script shell script in the examples folder it specifically says that it cannot find the WebSocket Variant in ServerTransports, once I added that feature to common the build_wasm script was able to compile the wasm examples.

Reply:
I am building again with it commented out so that I can screen shot the error.

Reply:
I see, i'm a little hesitant on including this dependency by default for the example.
Maybe the build_wasm script command can just use 
`--features=lightyear_examples_common/websocket`

(I haven't used the script myself, it was contributed by @RJ for bevygap, I just call `trunk server` to start the wasm examples)

Reply:
ahhh, that would work too! I will add another commit in a second.

Reply:
The error about the websockets was from running [./build_wasm.sh](https://github.com/cBournhonesque/lightyear/blob/main/examples/build_wasm.sh)

![2025-01-09_11-51-41](https://github.com/user-attachments/assets/4afcf347-544b-4190-8483-97d36a945223)

I will try out using `trunk server` like you do:

https://github.com/cBournhonesque/lightyear/tree/main/examples#testing-in-wasm-with-webtransport

Reply:
@cBournhonesque let me know if anything else needs fixed/changed. Thanks :)